timesketch/lib/testlib.py

```diff
@@ -248,6 +247,12 @@ def version(self):
         """
         return '6.0'
 
+    # pylint: disable=unused-argument
+    def search_stream(self, query_string, query_filter, query_dsl,
```

Comment:
In the previous implementation, the function sessionizer.event_stream() was mocked. It returns a generator of Event objects, that are recieved in JSON format from the ElasticsearchDataStore.search_stream() and them wrapped in Event objects. Instead of mocking the whole function, only the behaviour of the ElasticserachDataStore was mocked as implementing the same funciton in the MockDataStore class.